packs/prometheus/variables.hcl

```diff
@@ -109,6 +119,22 @@ scrape_configs:
 EOF
 }
 
+variable "prometheus_task_app_rules_yaml" {
+  description = "The Prometheus configuration to pass to the task."
+  type        = string
+  default     = <<EOF
+---
+groups:
+- name: AllInstances
+  rules:
+  - alert: PrometheusAlertmanagerE2eDeadManSwitch
+    expr: vector(1)
+    for: 0m
+    labels:
+      severity: critical
```

Comment:
What does this do and should this default to empty considering alertmanager is not guaranteed to be running?

Reply:
this is a check that will always be firing as a test/signal that prometheus is correctly configured and that it is able to deliver alerts to the alertmanager. I am happy to move it into a vars file similar to the jenkins job

Reply:
moved into separate file and mentioned this in the readme